store/tikv/region_request_test.go

```diff
@@ -255,10 +251,20 @@ func (s *testRegionRequestSuite) TestNoReloadRegionForGrpcWhenCtxCanceled(c *C)
 	region, err := s.cache.LocateRegionByID(s.bo, s.region)
 	c.Assert(err, IsNil)
 
-	_, err = sender.SendReq(s.bo, req, region.Region, 3*time.Second)
-	c.Assert(grpc.Code(errors.Cause(err)), Equals, codes.Canceled)
+	bo, cancel := s.bo.Fork()
+	cancel()
+	_, err = sender.SendReq(bo, req, region.Region, 3*time.Second)
+	c.Assert(errors.Cause(err), Equals, goctx.Canceled)
 	c.Assert(s.cache.getRegionByIDFromCache(s.region), NotNil)
 
+	// Just for covering error code = codes.Canceled.
+	client1 := &cancelContextClient{
+		Client:       newRPCClient(),
+		redirectAddr: addr,
+	}
+	sender = NewRegionRequestSender(s.cache, client1, kvrpcpb.IsolationLevel_SI)
+	sender.SendReq(s.bo, req, region.Region, 3*time.Second)
```

Comment:
This code is "Just for covering error code = codes.Canceled."
Not testing anything.